SimulationRuntime/c/util/simulation_options.h

```diff
@@ -144,6 +144,9 @@ enum _FLAG
   FLAG_STEADY_STATE_TOL,
   FLAG_UP_HESSIAN,
   FLAG_W,
+  FLAG_DATA_RECONCILE,
+  FLAG_DATA_RECONCILE_Sx,
+  FLAG_DATA_RECONCILE_Eps,
```

Comment:
The flags should be sorted in alphabetic order.

Reply:
@lochel , added the Flags in alphabetic order see https://github.com/OpenModelica/OMCompiler/pull/2845/commits/8af3b0933f715f6551edc2468f0559a9fd524667

Reply:
@lochel sorted the flags according to name